.github/workflows/ci.yml

```diff
@@ -1,35 +1,31 @@
 name: CI
-on: 
+on:
   pull_request:
```

Comment:
Can we add a workflow_dispatch trigger for the CI workflow so we have the ability to manually run the checks without a PR

Reply:
@sullan2 , I saw `macos_fix` branch has that trigger in the ci workflow. Should we wait until that branch is merged or reflect it right away here in this branch?

Reply:
@dreamer2368 It would be good to have the trigger now, for easier testing.